policy/scan/local_scanner.go

```diff
@@ -374,14 +375,20 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		}
 
 		// attach the asset details to the assets list
+		// also override asset name if flag --asset-name was used
 		for i := range assets {
 			log.Debug().Str("asset", assets[i].asset.Name).Strs("platform-ids", assets[i].asset.PlatformIds).Msg("update asset")
 			platformMrn := assets[i].asset.PlatformIds[0]
 			assets[i].asset.Mrn = platformAssetMapping[platformMrn].AssetMrn
 			assets[i].asset.Url = platformAssetMapping[platformMrn].Url
+			assetName := viper.GetString("asset-name")
```

Comment:
on the question of how to work with this when there is more than a single asset i think we should either:
1. ignore the flag in that case
2. only apply it to the primary asset, targeted for connection. we return an asset and an inventory from the connect call. we should probably only apply the flag to the asset and not the inventory

@chris-rock any thoughts?